Cargo.toml

```diff
@@ -27,6 +27,7 @@ wasmer-cache = { version = "2.0.0", path = "lib/cache", optional = true }
 wasmer-types = { version = "2.0.0", path = "lib/types", package = "wasmer-types-near" }
 wasmer-middlewares = { version = "2.0.0", path = "lib/middlewares", optional = true }
 cfg-if = "1.0"
+tracing-span-tree = "0.1"
```

Comment:
I guess its' better not to add this dependency, as we only use it for ad-hock benchmarking.